tutorial/part3.md

```diff
@@ -474,13 +474,13 @@ Finally in movement we need to check if the location we want to move to has an e
 
 +    // check for blockers
 +    const blockers = [];
-+    for (const e of ecs.entities.all) {
-+      if (e.position.x === mx && e.position.y === my && e.isBlocking) {
++    for (const e of world._entities.all) {
```

Comment:
this can now use the helper method https://github.com/ddmills/geotic/blob/master/src/World.js#L22:

```
for (const e of world.getEntities()) {
    ...
}
```

Reply:
adjusted my pr